internal/k8s-engine/graphql/domain/policy/converter.go

```diff
@@ -153,36 +152,36 @@ func (c *Converter) manifestRefsToGraphQL(in *[]types.ManifestRef) []*graphql.Ma
 	return out
 }
 
-func (c *Converter) typeInstancesToInjectToGraphQL(in []policy.TypeInstanceToInject) []*graphql.TypeInstanceReference {
-	var out []*graphql.TypeInstanceReference
+func (c *Converter) typeInstancesToInjectToGraphQL(in []policy.RequiredTypeInstanceToInject) []*graphql.RequiredTypeInstanceReference {
+	var out []*graphql.RequiredTypeInstanceReference
 
 	for _, item := range in {
-		out = append(out, &graphql.TypeInstanceReference{
-			ID:      item.ID,
-			TypeRef: c.manifestRefToGraphQL(item.TypeRef),
+		out = append(out, &graphql.RequiredTypeInstanceReference{
+			ID:          item.ID,
+			Description: item.Description,
```

Comment:
WDYT about resolving that TypeRef already here?

Currently after applying:
```yaml
cat > /tmp/aws-policy.yaml << ENDOFFILE
rules:
  - interface:
      path: cap.interface.containerization.kubernetes.deploy
    oneOf:
      - implementationConstraints:
          path: cap.implementation.aws.containerization.rke2.deploy
        inject:
          requiredTypeInstances:
          - id: "e7f9f8ee-2217-4704-af34-2610727ec567"
            typeRef:
              path: "cap.type.aws.auth.credentials"
              revision: "0.1.0"

  - interface:
      path: cap.*
    oneOf:
      - implementationConstraints:
          requires:
            - path: "cap.core.type.platform.kubernetes"
      - implementationConstraints: {}
ENDOFFILE

capact policy apply -f /tmp/aws-policy.yaml
```

We get:
```yaml
$ capact pol get
rules:
- interface:
    path: cap.interface.containerization.kubernetes.deploy
  oneOf:
  - implementationConstraints:
      path: cap.implementation.aws.containerization.rke2.deploy
    inject:
      requiredTypeInstances:
      - id: e7f9f8ee-2217-4704-af34-2610727ec567
- interface:
    path: cap.*
  oneOf:
  - implementationConstraints:
      requires:
      - path: cap.core.type.platform.kubernetes
  - implementationConstraints: {}
```

but IMO it will be nice to see the typeRef too, as we resolve it anyway